orangecontrib/text/widgets/owpreprocess.py

```diff
@@ -620,6 +620,7 @@ def commit(self):
             self.send(Output.PP_CORPUS, None)
 
     def apply(self):
+        self.set_minimal_width()
```

Comment:
@s-alexey this still doesn't quite work. The problem is that when I have no data on input (or when the auto-commit is disabled) clicking on section on/off buttons does not trigger the resize instantly. The widget would only resize once I click apply or once I provide the data. To reproduce, clean the widget settings, open canvas, put on the Preprocess Text widget and enable POS Tagger section. I get this (see how ON/OFF buttons are outside the visible part):

![screen shot 2016-12-05 at 11 34 38](https://cloud.githubusercontent.com/assets/713026/20882116/ae580ea4-badf-11e6-9868-fee768674069.png)


IMO we should move `self.set_minimal_width()` from `apply` method (which is only called when the data is present) to `settings_invalidated` method which is always called after some change occurs in the child widgets.